kong/runloop/handler.lua

```diff
@@ -705,7 +828,7 @@ local function balancer_setup_stage1(ctx, scheme, host_type, host, port,
 end
 
 
-local function balancer_setup_stage2(ctx)
+local function balancer_execute(ctx)
```

Comment:
Regardless of whether this is a good change or not, why are these balancer functions being renamed (with no functionality change) in this PR? This seems totally unrelated.

Reply:
@kikito, feel free to extract that to other pr.

Reply:
Ok, extracted that commit as well as the localization to #4636